agent/job_runner.go

```diff
@@ -79,9 +80,18 @@ func (r JobRunner) Create() (runner *JobRunner, err error) {
 		return nil, err
 	}
 
+	args, err := shell.Parse(r.AgentConfiguration.BootstrapScript)
```

Comment:
This moved up here because I moved it out of process.Process so that the error can be handled better.